### PR TITLE
feat: Range components accessibility

### PR DESCRIPTION
**Pull request by jyash97**

### CODE REVIEW

File: packages/web/src/components/range/MultiRange.js [hunk 1]
```diff
@@ -240,16 +240,23 @@ class MultiRange extends Component {
...
+					role="listbox"
+					aria-label={`${this.props.componentId}-items`}
+				>
 					{this.props.data.map(item => (
 						<li
 							key={item.label}
 							className={`${this.state.selectedValues[item.label] ? 'active' : ''}`}
+							role="option"
+							aria-checked={!!this.state.currentValue[item.label]}
+							aria-selected={!!this.state.currentValue[item.label]}
```
Comment (bietkul) [inline packages/web/src/components/range/MultiRange.js]: Can you keep a variable to store `!!this.state.currentValue[item.label]`, and use it further.

**Review by bietkul — APPROVED** [review]

File: packages/web/src/components/range/MultiRange.js (hunk 1 above)
Comment (bietkul) [reply packages/web/src/components/range/MultiRange.js]: @jyash97 Can you update it?

File: packages/web/src/components/range/MultiRange.js [hunk 2]
```diff
@@ -240,29 +240,43 @@ class MultiRange extends Component {
...
+					aria-label={`${this.props.componentId}-items`}
+				>
+					{this.props.data.map((item) => {
+						const isChecked = !!this.state.selectedValues[item.label];
+						return (
+							<li
+								key={item.label}
+								className={`${
+									this.state.selectedValues[item.label] ? 'active' : ''
+								}`}
```
Comment (bietkul) [inline packages/web/src/components/range/MultiRange.js:255]: @jyash97 What's the point of adding a variable if you're not using it.

**Review by bietkul — CHANGES_REQUESTED** [review]

File: packages/web/src/components/range/MultiRange.js (hunk 2 above)
Comment (jyash97) [reply packages/web/src/components/range/MultiRange.js:255]: Missed this line will fix it

**Pull request merged**